src/policy/fees.cpp

```diff
@@ -886,8 +902,14 @@ bool CBlockPolicyEstimator::Write(CAutoFile& fileout) const
     return true;
 }
 
-bool CBlockPolicyEstimator::Read(CAutoFile& filein)
-{
+bool CBlockPolicyEstimator::ReadEstFile() {
```

Comment:
You don't need a separate function for this since it's only called in one place. You can just insert this code into the constructor.